examples/fabric-admin/device_manager/DeviceSynchronization.cpp

```diff
@@ -142,13 +142,19 @@ void DeviceSynchronizer::OnReportEnd()
 
 void DeviceSynchronizer::OnDone(app::ReadClient * apReadClient)
 {
+    ChipLogProgress(NotSpecified, "Synchronization complete for NodeId:" ChipLogFormatX64, ChipLogValueX64(mNodeId));
+
 #if defined(PW_RPC_ENABLED)
-    if (mState == State::ReceivedResponse && !DeviceManager::Instance().IsCurrentBridgeDevice(mNodeId))
+    // We need to register the remote bridge device as a bridged device on the local bridge. This ensures that the ecosystem
+    // triggering the fabric sync process is notified of the successful reverse commissioning.
+    if (mState == State::ReceivedResponse && (mIsReverseCommissioning || !DeviceManager::Instance().IsCurrentBridgeDevice(mNodeId)))
     {
+        mIsReverseCommissioning = false;
```

Comment:
Why add this flag if we can ask `DeviceManager` if the node id is the remote bridge? (Note if that functionality is not there we can add it in).

I also don't understand why we want to enumerate the remote bridge as an endpoint on the local bridge. What part of the spec are you trying to implement here. If you are not implementing something from the spec, what is missing from the spec that you are trying to address here?

Reply:
See, the reason above, the current TH needs this and current implementation did that,  this change is to keep backward compatibility

Reply:
Per offline talking, we plan to update the current cert tests to not using the reverse commission to commission the test server, we need to hole this PR until we update all relevant cert tests